### PR TITLE
[TASK-1] Shoe Object Modelling

### PR DESCRIPTION
**Pull request by SaraiNina**
Am facut push de pe local pe remote, apoi am facut pull reguest pe master

### CODE REVIEW

File: com/nina/modelling/shoes/KidsShoe.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.nina.modelling.shoes;
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/KidsShoe.java]: Please rename pull request to respect format task name and some general meaningful name. for example: [TASK-1] Shoe Object Modelling

File: com/nina/modelling/shoes/KidsShoe.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+public class KidsShoe extends Shoe {
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/KidsShoe.java]: KidShoe

File: com/nina/modelling/shoes/KidsShoe.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    public KidsShoe(String category){
+        this.category = category;
+    }
+    public KidsShoe(String accessories, String type){
+        this.accessories = accessories;
+        this.type = type;
+    }
+
+    public String getCategory(){
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/KidsShoe.java]: In this case better remove word get from method names

File: com/nina/modelling/shoes/Shoe.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+   }
+   public Shoe(String model, String closingType) {
+       this.closingType = closingType;
+       this.closingType = model;
+   }
+   public Shoe(String material){
+       this.material = material;
+   }
+public  Shoe(int size){
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/Shoe.java]: Format code folding style for whole classes. Code->Reformat Code

File: com/nina/modelling/shoes/ShoeFactory.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package com.nina.modelling.shoes;
+
+public class ShoeFactory extends Shoe {
+
+    public static void main(String[] args) {
+
+        KidsShoe kidsShoe = new KidsShoe("Summer");
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/ShoeFactory.java]: We need to create objects for each type of constructor and try to instantiate all fields defined in classes.

File: com/nina/modelling/shoes/ShoeFactory.java [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        kidsShoe.getAccessories();
+        MenShoe menShoe = new MenShoe("Ama");
+        MenShoe menShoe1 = new MenShoe();
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/ShoeFactory.java]: Bad practice to name variables Student1 and Strudent2, please rename to studentAna and studentIon or just ana and ion.

File: com/nina/modelling/shoes/ShoeFactory.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
+package com.nina.modelling.shoes;
+
+public class ShoeFactory extends Shoe {
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/ShoeFactory.java]: Please read clean code rules and apply: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

File: src/main/java/com/nina/modelling/shoes/KidsShoe.java [hunk 8]
```diff
@@ -2,28 +2,28 @@
 
 public class KidsShoe extends Shoe {
```
Comment (alexandru-corghencea) [inline src/main/java/com/nina/modelling/shoes/KidsShoe.java]: All class names and variables, comments, commits, branches and even github comments should be in English.

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

File: com/nina/modelling/shoes/KidsShoe.java (hunk 1 above)
Comment (SaraiNina) [reply com/nina/modelling/shoes/KidsShoe.java]: I rename pull request in english.

File: com/nina/modelling/shoes/KidsShoe.java (hunk 2 above)
Comment (SaraiNina) [inline com/nina/modelling/shoes/KidsShoe.java]: I changed name of class KidShoe

File: src/main/java/com/nina/modelling/shoes/KidsShoe.java (hunk 8 above)
Comment (SaraiNina) [reply src/main/java/com/nina/modelling/shoes/KidsShoe.java]: Ok

File: com/nina/modelling/shoes/KidsShoe.java (hunk 3 above)
Comment (SaraiNina) [reply com/nina/modelling/shoes/KidsShoe.java]: I change the name of methods in class KidShoe

File: com/nina/modelling/shoes/ShoeFactory.java (hunk 5 above)
Comment (SaraiNina) [reply com/nina/modelling/shoes/ShoeFactory.java]: I created objects for each type of constructors and instantiated it.

File: com/nina/modelling/shoes/ShoeFactory.java (hunk 6 above)
Comment (SaraiNina) [reply com/nina/modelling/shoes/ShoeFactory.java]: I know

File: com/nina/modelling/shoes/Shoe.java (hunk 4 above)
Comment (SaraiNina) [reply com/nina/modelling/shoes/Shoe.java]: I format cod for  all classes

File: com/nina/modelling/shoes/ShoeFactory.java (hunk 7 above)
Comment (SaraiNina) [reply com/nina/modelling/shoes/ShoeFactory.java]: I read about clean code

File: src/main/java/com/nina/modelling/shoes/MenShoe.java [hunk 9]
```diff
@@ -2,17 +2,19 @@
 
 public class MenShoe extends Shoe {
```
Comment (alexandru-corghencea) [inline src/main/java/com/nina/modelling/shoes/MenShoe.java]: ManShoe

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

File: Task 1 - Shoe Object Modelling.txt [hunk 10]
```diff
@@ -0,0 +1,5 @@
+Object Modeling Shoes:
```
Comment (alexandru-corghencea) [inline Task 1 - Shoe Object Modelling.txt]: Move the file into resource folder
https://stackoverflow.com/questions/25786185/what-is-the-purpose-for-the-resource-folder-in-maven

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

File: com/nina/modelling/shoes/KidsShoe.java (hunk 1 above)
Comment (alexandru-corghencea) [reply com/nina/modelling/shoes/KidsShoe.java]: test

**Comment by SaraiNina** [comment]
SAL

File: src/main/java/com/nina/modelling/shoes/MenShoe.java (hunk 9 above)
Comment (SaraiNina) [reply src/main/java/com/nina/modelling/shoes/MenShoe.java]: Written a correct name

File: Task 1 - Shoe Object Modelling.txt (hunk 10 above)
Comment (SaraiNina) [reply Task 1 - Shoe Object Modelling.txt]: A resource folder was added .

**Review by SaraiNina — COMMENTED** [review]
Changes executed

File: src/main/java/com/nina/modelling/shoes/ShoeFactory.java [hunk 11]
```diff
@@ -6,34 +6,33 @@ public class ShoeFactory extends Shoe {
...
         Shoe shoe = new Shoe();
         Shoe shoe1 = new Shoe("Skin", "Zipper");
         Shoe shoe2 = new Shoe("Skin");
         System.out.println(" Shoe ClosingType = " + shoe.getClosingType());
         System.out.println(" Shoe Material = " + shoe1.getMaterial() + "" + shoe2.getMaterial());
+        System.out.println(" Shoe Model = " + shoe.getModel());
 
 
 
         WomanShoe womanShoe1 = new WomanShoe("Flower", "Zar");
```
Comment (alexandru-corghencea) [inline src/main/java/com/nina/modelling/shoes/ShoeFactory.java]: No variables of womanShoe 1,2,3
needs better name

File: src/main/java/com/nina/modelling/shoes/KidShoe.java [hunk 12]
```diff
@@ -11,21 +11,24 @@ public KidShoe(String category, String accessory, String type) {
...
         return category;
     }
-    public String getAccessory(){
+
+    public String Accessory() {
         return accessory;
     }
-    public String getType(){
+
+    public String Type() {
```
Comment (alexandru-corghencea) [inline src/main/java/com/nina/modelling/shoes/KidShoe.java]: If you changed these getters to naming without get, make this change for all getters in the code.

File: com/nina/modelling/shoes/ShoeFactory.java (hunk 6 above)
Comment (alexandru-corghencea) [reply com/nina/modelling/shoes/ShoeFactory.java]: So change it please

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

File: com/nina/modelling/shoes/MenShoe.java [hunk 13]
```diff
@@ -0,0 +1,18 @@
+package com.nina.modelling.shoes;
+
+public class MenShoe extends Shoe {
```
Comment (alexandru-corghencea) [inline com/nina/modelling/shoes/MenShoe.java]: Objects should be in singular mode

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]
All commit messages, objects and filenames should respect the format and language

File: src/main/java/com/nina/modelling/shoes/KidShoe.java [hunk 14]
```diff
@@ -24,11 +24,11 @@ public String category() {
         return category;
     }
 
-    public String Accessory() {
+    public String accessory() {
         return accessory;
```
Comment (alexandru-corghencea) [inline src/main/java/com/nina/modelling/shoes/KidShoe.java:28]: All commit messages, objects and filenames should respect the format and language

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 15]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (alexandru-corghencea) [inline pom.xml:2]: test comment thread

**Review by alexandru-corghencea — CHANGES_REQUESTED** [review]

**Pull request closed without merging**